ducktape/cluster/remoteaccount.py

```diff
@@ -90,6 +89,14 @@ def kill_process(self, process_grep_str, clean_shutdown=True, allow_fail=False):
             self.ssh(cmd, allow_fail)
 
     def scp_from_command(self, src, dest, recursive=False):
+        if type(src) == list:
+            if len(src) == 0:
+                src = ""
+            elif len(src) == 1:
+                src = src[0]
+            else:
+                src = "\{" + ",".join(src) + "\}"
```

Comment:
Cool! I was unaware of this in bash!

One potential drawback is for people who aren't using bash as their shell. Not sure if we want to worry about that now, but it may become a problem.